### DISCUSSION TITLE
How to show help message if no command is used?

### DISCUSSION THREAD

**Discussion by winston0410** [c4spar/cliffy#489] [Q&A] [answered]
Is it possible to show help message, if no command or `--help` is used? For example, 
`deno run https://deno.land/x/cliffy@v0.25.4/examples/command/help.ts` will not log anything right now.

**Comment by c4spar** [comment, marked as answer]
You can use `cmd.showHelp()` or `cmd.getHelpt()` to print or get the help output of the specified command as described hrere: https://github.com/c4spar/deno-cliffy/discussions/179.

**Reply by winston0410** [reply]
Found exactly what I wanted, thanks!
